uber/sep_commands.py

```diff
@@ -172,3 +172,25 @@ def reset_uber_db():
     assert c.DEV_BOX, 'reset_uber_db is only available on development boxes'
     Session.initialize_db(modify_tables=True, drop=True)
     insert_admin()
+
+
+
+@entry_point
+def decline_and_convert_dealer_groups():
```

Comment:
I realized that it would take too long to iterate over hundreds of groups using the web interface.

I've wrapped the functionality in a sep command. I also realize that means the main `_decline_and_convert_dealer_group()` function is probably declared in the wrong place, but I don't really have time to "do it right".